src/watchmaker/static/shell/bootstrappers/watchmaker-bootstrap.ps1

```diff
@@ -57,6 +57,23 @@ function Install-Python-MSI {
   Start-Process "msiexec.exe" -ArgumentList ${Arguments} -Wait
 }
 
+function Install-Python-Packages {
```

Comment:
Makes sense to me - I had wondered about setting it up that way so anyone can add more python modules as they see fit.